src/components/QuranReader/ReadingView/Line.tsx

```diff
@@ -40,13 +50,20 @@ const Line = ({ lineKey, words, isBigTextLayout, pageIndex, lineIndex }: LinePro
   const firstWordData = getWordDataByLocation(words[0].location);
   const shouldShowChapterHeader = firstWordData[1] === '1' && firstWordData[2] === '1';
 
+  const { quranFont, quranTextFontScale } = useSelector(
+    selectQuranReaderStyles,
+    shallowEqual,
+  ) as QuranReaderStyles;
+
   return (
     <div
       ref={selectedItemRef}
       id={lineKey}
       className={classNames(styles.container, {
         [styles.highlighted]: isHighlighted,
         [styles.mobileInline]: isBigTextLayout,
+        [styles[getLineWidthClassName(quranFont, quranTextFontScale)]]: isFixedWidth,
+        [getLineWidthClassName(quranFont, quranTextFontScale)]: true,
```

Comment:
2 things: 

1. any reason why we are having this without `styles`? and if we do, it's a duplicate of the previous line
2. if it's always `true`, we should make it as a param to `classNames(` instead